src/opentype/tables/advanced/shared/subtables/lookuptypes/gsub-2.js

```diff
@@ -0,0 +1,25 @@
+import { LookupType } from "./gsub.js";
+
+export class LookupType2 extends LookupType {
```

Comment:
Just for ease of reading, I prefer to use a single `export` statement at the end of the file rather than exporting each thing "with its declaration", so:
```
import { LookupType } from "./gsub.js";

class LookupType2 ....

class SequenceTable ...

export { LookupType2 }
```